lib/src/model.ts

```diff
@@ -141,7 +158,7 @@ export class Model extends EventTarget {
         event: RecognizerEvent,
         listener: (message: RecognizerMessage) => void
       ) {
-        model.addEventListener(event, (event: any) => {
+        this.addEventListener(event, (event: any) => {
```

Comment:
If you forward events from the recognizer now I think you don´t need to check for `event.detail` and `ServerMessage.isRecognizerMessage` anymore because this has been done in the model already, hasn´t it?

Reply:
True